python/openinference-instrumentation/pyproject.toml

```diff
@@ -40,10 +40,39 @@ Homepage = "https://github.com/Arize-ai/openinference/tree/main/python/openinfer
 path = "src/openinference/instrumentation/version.py"
 
 [tool.hatch.build.targets.sdist]
-include = [
-  "/src",
-  "/tests",
+exclude = [
+  "src/openinference/instrumentation/*/*",
 ]
```

Comment:
I'm a little confused, doesn't the exclude also need to be in each instrumentation library that depends on `openinference-instrumentation`?

Reply:
Resolved offline.